translations/messages-en.properties

```diff
@@ -888,4 +888,8 @@ contact.profile.delivery_code.miscarriage = Miscarriage
 contact.profile.delivery_code.f = Facility birth
 contact.profile.delivery_code.s = Home birth with skilled attendant
 contact.profile.delivery_code.ns = Home birth without skilled attendant
-contact.profile.delivery_code.unknown = Delivery
\ No newline at end of file
+contact.profile.delivery_code.unknown = Delivery
+sync.status.in_progress=Currently syncing…
+sync.status.not_required=All reports synced
```

Comment:
Good question.  I'm not sure if there was specific thinking behind this wording from the design team or not.

I agree that _docs_ is technically more correct, but perhaps _reports_ is more relevant to a user - he'll may be worrying whether the things he has submitted via the _Reports_ tab have been sent to the server.  Although... the _Reports_ tab is normally hidden, and he'll generally filling out reports via _Contacts_ or _Tasks_...

N.B. currently the only use of the word _docs_ in user-facing text is in the _About_ tab's db debug info, and soon to be in the boot screen...